.github/workflows/main.yml

```diff
@@ -13,9 +13,10 @@ jobs:
           - macos-latest
         version:
           - 1.34.0
-          - latest
+          # Updating this from latest to 1.42.0 for now as the "latest tag" (in this case 1.42.1) doesn't have the proper binaries available for download
+          - 1.42.0
```

Comment:
This defeats the purpose of the tests, which was to ensure that both a specific version, and an abstract one like `latest` worked. Let's see if we can figure out why binaries aren't published or if something is changing.

Reply:
@learoyklinginsmith if you back this change out, I'll merge past the missing binaries problem (I hope). Definitely worth getting the node version bumped.

Reply:
Will do!